src/ontology/mondo-edit.obo

```diff
@@ -413861,6 +413877,50 @@ name: obsolete histidinuria-renal tubular defect syndrome
 is_obsolete: true
 replaced_by: MONDO:0009346
 
+[Term]
+id: MONDO:0026762
+name: wieacker-wolff syndrome, female-restricted
+synonym: "WIEACKER-WOLFF SYNDROME, FEMALE-RESTRICTED; WRWFFR" RELATED [OMIM:301041]
+xref: OMIM:301041 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+is_a: MONDO:0025445 {source="OMIM:301041"} ! Wieacker-Wolff syndrome (spectrum)
+
+[Term]
+id: MONDO:0026763
+name: holoprosencephaly 13, x-linked
+synonym: "HOLOPROSENCEPHALY 13, X-LINKED; HPE13" RELATED [OMIM:301043]
+xref: OMIM:301043 {source="MONDO:equivalentTo"}
+xref: Orphanet:220386 {source="OMIM:301043"}
+xref: Orphanet:93925 {source="OMIM:301043"}
+is_a: MONDO:0003847 ! Mendelian disease
```

Comment:
I made holoprosencephaly 13, x-linked a child of holoprosencephaly

I didn't add an equiv axiom because STAG2 is an individual and not a class. I don't understand this.

Reply:
I created a ticket for Ada: https://github.com/monarch-initiative/mondo/issues/2223